lib/samples/my_awesome_consumer.rb

```diff
@@ -11,6 +11,5 @@ class MyAwesomeConsumer
   def perform(message, metadata)
     puts "Received message #{message} from broker - retry #{metadata[:retry_count]}"
     ## Fill with our logic here
-    raise StandardError
```

Comment:
@jleber This was the remnant of a previous test or something, right? This made the README instructions fail :sweat_smile:

Reply:
😬  yes, you're right (my fault, sorry for that)

Reply:
Sorry, you just broke a test, I broke production :sunglasses: :nail_care: